types/jsftp/jsftp-tests.ts

```diff
@@ -0,0 +1,25 @@
+import Ftp from 'jsftp';
```

Comment:
You should use `import name = require('package');` instead.

If you still want to use the syntax `import name from 'package';` you need to use `esModuleInterop` on the compiler settings from your consuming code.